src/lib.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![cfg_attr(feature = "nightly", feature(doc_auto_cfg))]
+#![cfg_attr(feature = "nightly", feature(doc_cfg))]
```

Comment:
Why was this changed?

Reply:
`doc_auto_cfg` has been replaced by `doc(auto_cfg)` with is feature gated behind `doc_cfg` feature. See the RFC here: https://github.com/rust-lang/rfcs/blob/master/text/3631-rustdoc-cfgs-handling.md
Not including `doc(auto_cfg)` was a mistake which I've now fixed: https://github.com/RReverser/wasmbin/pull/7/commits/bb971ced5aea4f723244eb67000c1d6f9066fd12

Reply:
Ah, that makes more sense, thanks.